sources/3.0.0/FC/S-102_Ed_3.0.0_FC.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="utf-8"?>
+<S100FC:S100_FC_FeatureCatalogue xmlns:S100FC = "http://www.iho.int/S100FC/5.2" xmlns:S100Base="http://www.iho.int/S100Base/5.0" xmlns:S100CI="http://www.iho.int/S100CI/5.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:S100CD="http://www.iho.int/S100CD/5.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<S100FC:name>S-102</S100FC:name>
+	<S100FC:scope>Global</S100FC:scope>
+	<S100FC:fieldOfApplication>Marine Navigation</S100FC:fieldOfApplication>
+	<S100FC:versionNumber>3.0.0</S100FC:versionNumber>
+	<S100FC:versionDate>2024-05-31</S100FC:versionDate>
+	<S100FC:productId>S-102</S100FC:productId>
+	<S100FC:producer>
+		<S100CI:role>pointOfContact</S100CI:role>
+		<S100CI:party>
+			<S100CI:CI_Organisation>
+				<S100CI:name>International Hydrographic Organization</S100CI:name>
+				<S100CI:contactInfo>
+					<S100CI:phone>
+						<S100CI:number>+377 93 10 81 00</S100CI:number>
+						<S100CI:numberType>voice</S100CI:numberType>
+					</S100CI:phone>
+					<S100CI:phone>
+						<S100CI:number>+377 93 10 81 40</S100CI:number>
+						<S100CI:numberType>fax</S100CI:numberType>
+					</S100CI:phone>
+					<S100CI:address>
+						<S100CI:administrativeArea>4b quai Antoine 1er</S100CI:administrativeArea>
+						<S100CI:country>Principauté de Monaco</S100CI:country>
+						<S100CI:electronicMailAddress>info@iho.int</S100CI:electronicMailAddress>
+					</S100CI:address>
+					<S100CI:onlineResource>
+						<S100CI:linkage>www.iho.int</S100CI:linkage>
+					</S100CI:onlineResource>
+		</S100CI:contactInfo>
```

Comment:
It doesn't matter, the catalog is not intended to be read by a human. From my point of view, it would even make sense to completely remove the indentation, as this saves storage space.

Reply:
I added the indents. Thank you both!